mmengine/dist/utils.py

```diff
@@ -537,3 +544,137 @@ def cast_data_device(
     else:
         raise TypeError('data should be a Tensor, list of tensor or dict, '
                         f'but got {data}')
+
+
+def launch(
+        main_func: Callable,
+        num_proc_per_node: Union[list, int] = 1,
+        num_nodes: int = 1,
+        node_rank: int = 0,
+        master_addr: str = '127.0.0.1',
+        master_port: str = 'auto',
+        args: tuple = (),
+) -> None:
```

Comment:
```suggestion
def launch(
    func: Callable,
    num_proc_per_node: Union[list, int] = 1,
    num_nodes: int = 1,
    node_rank: int = 0,
    master_addr: str = '127.0.0.1',
    master_port: str = 'auto',
    args_to_func: tuple = (),
) -> None:
```

Reply:
Refer to

https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process
https://pytorch.org/docs/stable/multiprocessing.html#torch.multiprocessing.spawn
![image](https://user-images.githubusercontent.com/57566630/220991844-2b0f3546-2180-4ef5-8cdc-2bd1442c8194.png)

Should we rename `args` to `args_to_func`?